docs/Developers/Review.rst

```diff
@@ -65,6 +65,14 @@ The review process for coala is as follows:
    ``corobo mark wip <URL>`` or ``corobo mark pending <URL>``.
 3. The developers will acknowledge the commits by writing
 
+   * In case a member is reviewing it:
+
+    - ``Looks good to me`` better known as ``LGTM`` in case the commit is
+      ready, or
+    - Additional comments if something needs to be rectified.
```

Comment:
Remove Additional, it's either LGTM or it's not.

Reply:
I think it's not like that. Members do provide additional information regarding the required changes to be made.

Reply:
Oh sorry, I got it now. Thanks for it 🎉